source/indexes/single-field-index.txt

```diff
@@ -0,0 +1,109 @@
+.. _cpp-single-field-index:
+
+====================
+Single Field Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Single field indexes** are indexes that hold a reference to a single field of a
+document in a collection. These indexes improve single field query and
+sort performance. They also support :manual:`TTL Indexes </core/index-ttl>`
+that automatically remove documents from a collection after a certain
+amount of time or at a specified clock time.
+
+To create a single-field index, call the ``create_index()`` method and specify a document containing:
```

Comment:
S: stay consistent with either one word (single-field) or two (single field). Other pages are inconsistent with this, but seems like most use two words.

Also, same suggestion as above about using a full sentence to introduce a list:
```suggestion
To create a single field index, call the ``create_index()`` method and specify a document containing the following information:
```

Reply:
applies to all mentions of "single-field"